### DISCUSSION TITLE
Icons arrangement (thoughts)

### DISCUSSION THREAD

**Discussion by du82** [status-im/status-legacy#16636] [General]
Just thinking out loud: 
Shouldn't the chat icon be on the left, since the user will likely prefer to land on the 1:1 chat page when opening the app?

![image](https://user-images.githubusercontent.com/65834069/175842274-2fdbb1bc-abc0-4051-a58f-f834af07461f.png)

**Comment by ibrkhalil** [comment]
I totally agree, Icons should be sorted in order of importance to the user .. And this is a chat app.
